### DISCUSSION TITLE
Reset mod home screen

### DISCUSSION THREAD

**Discussion by CURBIMC** [Luke100000/minecraft-comes-alive#356] [Q&A] [answered]
Hello, I have a question, I am on the server and when starting, many players marked the Villager option, the questions is "" what is the configuration to reset and that the players see the home screen again and mark the Player option .  I am looking forward to your response .  Thanks a lot.

**Comment by CDAGaming** [comment, marked as answer]
`allowDestinyCommandOnce` and `allowDestinyCommandMoreThanOnce` are the config options related to this, though I have plans to make this into a simplified config option in 7.4
